src/scripts/app/context-menu.js

```diff
@@ -347,6 +352,17 @@ const _getItemContextOptions = function(item) {
     }
   );
 
+  if(game.modules.get("lets-trade-5e")?.active) {
+    if(!['feat','background','class','subclass','spell'].includes(item.type)) {
+      options.push({
+          name: `${game.i18n.localize("LetsTrade5E.Send")}`,
+          icon: `<i class="fas fa-balance-scale-right"></i>`,
+          callback: ()=>{
+            game.modules.get("lets-trade-5e").api.openItemTrade(item.actor?.id, item.id);
+          }
+      });
+    }
+  }
```

Comment:
Let the Let's Trade 5e module handle it, don't need to add integration this way.

Reply:
![image](https://user-images.githubusercontent.com/37833135/214440146-ae320c23-c80c-4401-b317-95055d7e613e.png)
Otherwise we will have duplicates like these.